spec/src/main/asciidoc/metrics_spec.adoc

```diff
@@ -1028,7 +1055,8 @@ annotated objects change.
 * The implementation must expose the values of the objects registered in the MetricsRegistry via REST-API as described
 in <<rest-api>>.
 * Metrics registered via non-annotations API need their values be set via updates from the application code.
-* The implementation must flag duplicate metrics upon registration and reject the duplicate.
+* The implementation must flag duplicate metrics upon registration and reject the duplicate unless the metric
+is explicitly marked as reusable upon first registration and in all subsequent registrations.
```

Comment:
It's not clear from this text if impl should throw an exception or just quietly not register the metric.  This is a case where you'd probably want to fail clearly and fast, so I'd advocate for throwing a runtime exception, such as IllegalArgumentException.

Also, it's worded as if at registration time you know if **ALL** subsequent registrations will be consistent with use of reusable -- which isn't knowable.

Reply:
Yes. It was not explicit before, so it makes sense to do that now.